docs/book/src/topics/capibmadm/powervs/port.md

```diff
@@ -0,0 +1,24 @@
+## PowerVS Network Commands
+
+### 1. capibmadm powervs port delete
+
+#### Usage:
+Delete PowerVS network port.
+
+#### Environmental Variable:
+IBMCLOUD_API_KEY: IBM Cloud API key.
+
+#### Arguments:
+--service-instance-id: PowerVS service instance id.
+
+--zone: PowerVS zone.
+
+--port-id: ID of network port.
+
+--network: Network ID or Name(preference will be given to the ID over Name).
```

Comment:
```suggestion
--network: Network ID or Name.
```
`Get()` function on network seems to work with both name and ID, so preference of one over other is not needed.